_data/hosting_providers.json

```
@@ -2699,4 +2699,24 @@
     "reviewed": "2021.12.18",
     "note": "The website is in Polish."
   }
+    {
+    "name": "MyHost (NZ)",
+    "link": "https://myhost.nz/",
+    "category": "full",
+    "tutorial": "",
+    "announcement": "",
+    "plan": "https://myhost.nz/hosting/web-hosting",
+    "reviewed": "2022.09.16",
+    "note": ""
+  }
+  {
+    "name": "SiteHost",
+    "link": "https://sitehost.nz/",
+    "category": "full",
```

Based on
> I’ve got good news for anyone who is currently using our [Cloud Containers platform](https://sitehost.nz/hosting/cloud-containers-2) – you can easily add free SSL certificates to your websites through our control panel. We even have a handy article that [walks you through the steps](http://kb.sitehost.co.nz/cloud-containers-containers/ssl). If you’re on one of our other platforms [get in touch](https://sitehost.nz/contact) and we'll sort you out.

and our text from https://certbot.eff.org/hosting_providers

> Full HTTPS Support: Hosting providers that offer free Let’s Encrypt HTTPS certificates automatically for their shared hosting product. No need to do anything on your end if you use their service.
>
> Partial HTTPS support: Hosting providers that offer free Let’s Encrypt HTTPS certificates for their shared hosting product, but require configuration. You’ll need to follow a tutorial.

I think this should be "partial" because SSL support doesn't come by default out of the box.

```suggestion
    "category": "partial",
```

Otherwise, this PR LGTM.